README.md

```diff
@@ -18,6 +18,135 @@ to facilitate unit-testing of OpenTracing Java instrumentation.
+ 
+It's inconvenient to pass an active `Span` from function to function manually, so OpenTracing provides an `ActiveSpanSource` abstraction to grant access to an `ActiveSpan` and to pin and `capture()` it for re-activation in other execution contexts (e.g., in an async callback).
+
+Access to the active `Span` is straightforward:
+
+```
+io.opentracing.Tracer tracer = ...;
+...
+ActiveSpan span = tracer.activeSpan();
+if (span != null) {
+    span.log("...");
+}
+```
+
+### Starting a new Span
+
+The simplest case – which does not take advantage of `ActiveSpanSource` – looks like this:
+
+```
+io.opentracing.Tracer tracer = ...;
+...
+Span span = tracer.buildSpan("someWork").startManual();
+try {
+    // (do things / record data to `span`)
+} finally {
+    span.finish();
+}
+```
+
+Or, to take advantage of `ActiveSpanSource` and automatic intra-process propagation, like this:
+
+```
+io.opentracing.Tracer tracer = ...;
+...
+try (ActiveSpan activeSpan = tracer.buildSpan("someWork").startActive()) {
+    // Do things.
+    //
+    // If we create async work, `activeSpan.capture()` allows us to pass the `ActiveSpan` along as well.
+}
+```
+
+The above is semantically equivalent to the more explicit try-finally version:
+
+```
+io.opentracing.Tracer tracer = ...;
+...
+ActiveSpan activeSpan = tracer.buildSpan("someWork").startActive();
+try {
+    // Do things.
+} finally {
+    activeSpan.deactivate();
+}
+```
+
+
+**If there is an `ActiveSpan`, it will act as the parent to any newly started `Span`** unless the programmer invokes `ignoringActiveSpan()` at `buildSpan()` time, like so:
+
+```
+io.opentracing.Tracer tracer = ...;
+...
+ActiveSpan span = tracer.buildSpan("someWork").ignoringActiveSpan().startActive();
+```
+
+### Deferring asynchronous work
+
+Consider the case where a `Span`'s lifetime logically starts in one thread and ends in another. For instance, the intra-Span timing breakdown might look like this:
+
+```
+[ ServiceHandlerSpan                                 ]
+|-FunctionA-|-----waiting on an RPC------|-FunctionB-|
+            
+------------------------------------------------> time
+```
+
+The `"ServiceHandlerSpan"` is _active_ when it's running FunctionA and FunctionB, and inactive while it's waiting on an RPC (presumably modelled as its own Span, though that's not the concern here).
+
+**The `ActiveSpanSource` makes it easy to `capture()` the Span and execution context in `FunctionA` and re-activate it in `FunctionB`.** Note that every `Tracer` must also implement `ActiveSpanSource`. These are the steps:
+
+1. Start the `ActiveSpan` via `Tracer.startActive()` rather than via `Tracer.startManual()`; or, if the `Span` was already `startManual()`ed, call `ActiveSpanSource#makeActive(span)`. Either route will yield an `ActiveSpan` instance that encapsulates the `Span`.
```

Comment:
`startManual()`ed - started manually (better English, and clear from the context).

Alternatively

Start the `ActiveSpan` via `Tracer.startActive()`; or, if the `Span` was already started manually via `Tracer.startManual()`, call `ActiveSpanSource#makeActive(span)`.